source/change-streams/tests/change-streams.json

```diff
@@ -0,0 +1,402 @@
+          "collection": "test",
+          "commandName": "insert",
+          "arguments": [
+            {
+              "x": 1
+            }
+          ]
+        }
+      ],
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "aggregate": "test",
+              "pipeline": [
+                {
+                  "$changeStream": {}
+                }
+              ]
+            },
+            "command_name": "aggregate",
+            "database_name": "change-stream-tests"
+          }
+        }
+      ],
+      "result": {}
+    },
+    {
+      "description": "The server returns change stream responses in the specified ``ServerResponse`` format",
+      "minServerVersion": "3.6.0",
+      "target": "collection",
+      "topology": [
+        "replicaset"
+      ],
+      "operations": [
+        {
+          "database": "change-stream-tests",
+          "collection": "test",
+          "commandName": "insert",
+          "arguments": [
+            {
+              "x": 1
+            }
+          ]
+        }
+      ],
+      "expectations": [],
+      "result": {
+        "success": [
+          {
+            "_id": {},
+            "documentKey": {},
+            "operationType": "insert",
+            "ns": {
+              "db": "change-stream-tests",
+              "coll": "test"
+            },
+            "fullDocument": {
+              "x": {
+                "$numberInt": "1"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "description": "Executing a `watch` helper on a Collection results in notifications for changes to the specified collection",
+      "minServerVersion": "3.6.0",
+      "target": "collection",
+      "topology": [
+        "replicaset"
+      ],
+      "operations": [
+        {
+          "database": "change-stream-tests",
+          "collection": "test2",
+          "commandName": "insert",
+          "arguments": [
+            {
+              "x": 1
+            }
+          ]
+        },
+        {
+          "database": "change-stream-tests-2",
+          "collection": "test",
+          "commandName": "insert",
+          "arguments": [
+            {
+              "y": 2
+            }
+          ]
+        },
+        {
+          "database": "change-stream-tests",
+          "collection": "test",
+          "commandName": "insert",
+          "arguments": [
+            {
+              "z": 3
+            }
+          ]
+        }
+      ],
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "aggregate": "test",
+              "pipeline": [
+                {
+                  "$changeStream": {}
+                }
+              ]
+            },
+            "command_name": "aggregate",
+            "database_name": "change-stream-tests"
+          }
+        }
+      ],
+      "result": [
```

Comment:
This is backwards.  `result` should be a document and `success` should be an array.

Reply:
done